utils/jumpToPattern.ts

```diff
@@ -0,0 +1,114 @@
+import { Editor as CodeMirrorEditor } from "codemirror";
+import { EditorPosition } from "obsidian";
+import { shim as matchAllShim } from "string.prototype.matchall";
+
+// Polyfill for String.prototype.matchAll, in case it's not available (pre-ES2020)
+matchAllShim();
```

Comment:
I'm guessing we don't actually need this polyfill since users are probably using a version of Obsidian that runs on a post-2020 Electron/Chromium version